packages/wp-checkout/public/index.html

```diff
@@ -13,6 +13,10 @@
 			-moz-osx-font-smoothing: grayscale;
 			background: #F3F4F5;
 		}
+
+		*:focus {
```

Comment:
These styles will only affect this page, which is just a demo for a host page. Is this a style rule we will want for all our hosts? If so, we should include it somewhere in the files in `src/components`.

Reply:
note to self: maybe I should rename this directory from `public` to `demo` to make it more obvious what it does!

Reply:
oh right. Good point! I'll move it to the parent container.